README.md

```diff
@@ -1,21 +1,139 @@
 # conda-lock
 
-Conda lock is a lightweight library that can be used to generate fully reproducible lock files for conda environments.
+Conda lock is a lightweight library that can be used to generate fully reproducible lock files for [conda][conda]
+environments.
 
 It does this by performing multiple solves for conda targeting a set of platforms you desire lockfiles for.
```

Comment:
Do we need the work "multiple" there? The CLI can be invoked for a single platform/single solve, right?